src/features/commands/character/character-command.service.ts

```diff
@@ -19,14 +19,16 @@ export class CharacterCommandService {
     return CharacterService.getInstance()
       .getEntity(message.author.id)
       .then((characterFound): void => {
-        const boldCharacterName = MessageFormattingService.getInstance().format(
-          TextFormatsEnum.BOLD,
-          characterFound.name
-        );
-        void DisplayMessageService.getInstance().message(
-          message,
-          `Your character's name is: ${boldCharacterName}`
-        );
+        if (characterFound) {
+          const boldCharacterName = MessageFormattingService.getInstance().format(
+            TextFormatsEnum.BOLD,
+            characterFound.name
+          );
+          void DisplayMessageService.getInstance().message(

```

Comment:
You could return this Promise and add an else for the character found which returns a Promise.reject.
Changing the type of the method is required I think.

Reply:
Yes. Though, I did not use a useless else after return statement. So I've directly put the return Promise.reject after the if.